### PR TITLE
remove team button support

### PR DESCRIPTION
**Pull request by coury-clark**
Adds team support to the remove owner button.

## Test plan
CleanShot 2023-06-08 at 15 14 26

### CODE REVIEW

File: client/web/src/repo/blob/own/OwnerList.tsx [hunk 1]
```diff
@@ -149,6 +149,8 @@ export const OwnerList: React.FunctionComponent<OwnerListProps> = ({
                                 )
                             )
                             .map((ownership, index) => {
+                                if (ownership.owner.__typename === 'Team') {
+                                }
```
Comment (sashaostrikov) [inline client/web/src/repo/blob/own/OwnerList.tsx:153]: should it be removed?
```suggestion
```

**Review by sashaostrikov — APPROVED** [review]
NICE! :shipit::shipit::shipit:

File: client/web/src/repo/blob/own/OwnerList.tsx (hunk 1 above)
Comment (coury-clark) [reply client/web/src/repo/blob/own/OwnerList.tsx]: well, that's embarrassing, thanks!

File: client/web/src/repo/blob/own/RemoveOwnerButton.tsx [hunk 2]
```diff
@@ -1,21 +1,28 @@
 import React from 'react'
 
+import { MutationFunctionOptions } from '@apollo/client'
```
Comment (courier-new) [inline client/web/src/repo/blob/own/RemoveOwnerButton.tsx:3]:
```suggestion
```

**Pull request merged**